server/test/services/applicant/ApplicantServiceTest.java

```diff
@@ -2021,6 +2021,97 @@ public void relevantProgramsForApplicant_hiddenFromIndex() {
         .containsExactly(programDefinition.id());
   }
 
+  @Test
+  public void relevantProgramsForApplicant_ShowsDraftThatIsDisabled() {
```

Comment:
**thought:** I'm kind of surprised we'd want to show draft applications for a disabled program. What would happen if the user went to continue filling out the application? Wouldn't it show that the program is disabled?

Reply:
This is very good point. Yes we shouldn't show it. But I am checking the code to see where to add it. It doesn't look like we update the applications with the latest program version. Do you have an idea @dkatzz ?